views/partials/what_to_do.hbs

```diff
@@ -1,4 +1,5 @@
 <ol id="what-to-do">
+  <li>What to Do</li>
```

Comment:
Should we give this an `id=""` or `class=""` and try and avoid `#what-to-do li:first-of-type {}` in the CSS, or are there subtle differences?

Reply:
Yeah, or maybe even taken out of the list and set as a sub head? There should be an `<lh>` tag.

Reply:
Interesting, I’ve never even heard of the ‘<lh>’ tag.

Reply:
It was almost a thing like 10 years ago and then disappeared. I don't know why it was never made official.

Reply:
Also, in response to another review comment about the status of the images... after MUCH trying and failing to batch them into smaller, more reasonably sized assets (that still looked decent), I finally just copied and pasted from the add-on. Once I'm not so mired in google analytics grief I'll probably start trying to figure out a better way to approach them. Any thoughts/ideas are totally appreciated!